feature/aks-cluster-creation/vars.tf

```diff
@@ -0,0 +1,116 @@
+// -- [ VARS ] ------------------------------------------------------
+
+// If You want to use existing network and resource group, just update the two variables, if not, leave them empty:
+variable "existing_vnet_subnet_id" {
+  description = "Resource id of existing the Virtual Network subnet where You want to join Your AKS cluster"
+  default = ""
+}
+
+variable "existing_resource_group_name" {
+  description = "Name of the existing AKS cluster resource group where You want to join Your AKS cluster"
+  default = ""
+}
+
+// General variables:
+
+variable "prefix" {
+    description = "Prefix for resources's name"
+    default     = "ropu"
+}
+
+variable "location" {
+    description = "AKS cluster location"
+    default     = "North Central US"
+}
+
+variable "kubernetes_version" {
+  description = "Kubernetes version"
+  default     = "1.17.7"
+}
+
+variable "vnet_address_space" {
+  description = "Network address space"
+  default = ["10.1.0.0/16"]
+}
+
+variable "aks_address_prefix" {
+  description = "SubNetwork address space"
+  default = ["10.1.1.0/24"]
+}
+
+variable "public_ssh_key_path" {
+    description = "Public ssh key path"
+    default     = "/workspaces/epiphany/core/src/epicli/ssh/id_rsa.pub"
```

Comment:
should be something more like `~/.ssh/some_public_rsa.pub`

Reply:
True :) Fixed